content/docs/installation/kubectl.md

```diff
@@ -67,8 +67,9 @@ cert-manager using the procedure determined by how you installed.
 
 > **Warning**: Uninstalling cert-manager or simply deleting a `Certificate` resource can result in 
 > TLS `Secret`s being deleted if they have `metadata.ownerReferences` set by cert-manager.
-> You can disable this behavior by setting `-enable-certificate-owner-ref=false` before deleting
-> a `Certificate` or `cert-manager`.
+> By default in cert-manager 1.8+, `--enable-certificate-owner-ref` is set to `false`, which will ensure
+> that owner references are removed. 
```

Comment:
Suggestion:

You can control the whether owner references are added to `Secret`s using `--enable-certificate-owner-ref` controller flag. By default this flag is set to false, which means that no owner references are added. However, in cert-manager older than v1.8 changing the flag's value from true to false _did not_ result in existing owner references being removed. This behaviour was fixed in cert-manager v1.8.